src/main/java/internet/store/dao/jdbc/OrderDaoJdbcImpl.java

```diff
@@ -0,0 +1,151 @@
+package internet.store.dao.jdbc;
+
+import internet.store.dao.OrderDao;
+import internet.store.exception.DataProcessingException;
+import internet.store.lib.Dao;
+import internet.store.model.Order;
+import internet.store.model.Product;
+import internet.store.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class OrderDaoJdbcImpl implements OrderDao {
+
+    @Override
+    public List<Order> getUserOrders(Long userId) {
+        String query = "SELECT order_id FROM orders WHERE deleted = false AND user_id = " + userId;
```

Comment:
can we fetch all information about orders here(except products)?

Reply:
Yes we can but we have only userId info and flag "deleted". Why do we need this to fetch?